README.md

```diff
@@ -361,6 +368,29 @@ spec:
 
 if you do not provide these fields, a role will be created for your instance-group by the controller (will require IAM access).
 
+
+#### EC2 Instance placement
```

Comment:
We should probably avoid adding more examples to main readme.md, we also have a ticket to make it shorter #193 
Wonder if we should add a file docs/examples/EKS.md and place all the examples there?